web/src/engine/websites/MangaUpJapan_e2e.ts

```diff
@@ -0,0 +1,24 @@
+import { TestFixture, type Config } from '../../../test/WebsitesFixture';
+
+const config: Config = {
```

Comment:
No need for Config anymore. Same for config variable.

Use new test file syntax

``` ts
new TestFixture({ 
....tests data
}).AssertWebsite();
```

sample : https://github.com/manga-download/haruneko/blob/master/web/src/engine/websites/ACGN_e2e.ts